website/docs/advanced-topics/server-side-rendering.md

```diff
@@ -195,6 +195,49 @@ fn main() {
 Example: [simple_ssr](https://github.com/yewstack/yew/tree/master/examples/simple_ssr)
 Example: [ssr_router](https://github.com/yewstack/yew/tree/master/examples/ssr_router)
 
+## Single thread mode
+
+Yew supports single thread mode for server-side rendering by `yew::LocalServerRenderer`. This mode would work in a single thread environment like WASI.
+
+```rust
+// Build it by `wasm32-wasi` target
```

Comment:
Preview 2, perhaps? It's nightly only but maybe it's better to use that. I'm totally fine with making WASI examples to be nightly only in yew (no need to make the framework nightly only for it; the consumption is much better in p2)

Reply:
> Preview 2, perhaps? It's nightly only but maybe it's better to use that. I'm totally fine with making WASI examples to be nightly only in yew (no need to make the framework nightly only for it; the consumption is much better in p2)

WASI's multi-threading proposal is still not stable even in preview2. If it were not for compatibility with the multi-threaded version of WASI, at least currently requiring users to use nightly channels to build WASI would do more harm than good.

Of course, I still support the new proposal of using WASI as soon as possible. When WASI's multi-threading capabilities stabilize at some point in the future, I'll improve the code.

Reply:
Makes sense. We can tell users that they can use networking from p2 if they want but not force it upon them